assets/stylesheets/components/_global-header.scss

```diff
@@ -4,7 +4,11 @@
 
 .global-header--inner {
   @extend %content-block;
-  padding: 24px;
+  padding: ($baseline-grid-unit * 4);
```

Comment:
Just having had a look at the layout and we should possibly handle left/right padding within either the `%content-block` mixin or as part of a grid.